### DISCUSSION TITLE
Welcome to stable-diffusion Discussions!

### DISCUSSION THREAD

**Discussion by hlky** [hlky/stable-diffusion#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WSzP** [comment]
I'm Peter W. Szabo, got 15+years of experience in UX/front-end, now a Ph.D. candidate in machine learning, and the creator of [promptoMANIA](https://promptomania.com/). I can share some publications, and I also have a book on UX if you are interested (User Experience Mapping, Packt, 2017, available on Amazon).

**Reply by altryne** [reply]
Sir I think promptoMANIA is incredible! 
I added it to my awesome list https://github.com/altryne/awesome-ai-art-image-synthesis
I worked on a DALL-E extension, that injects prompts right into the DALL-E interface
https://github.com/altryne/dalle-prompt-extension
Of course, this was before... SD! 

I would love to collaborate with you, in any way you think is cool to make promptoMANIA more accessible (maybe a browser extension?) 
😁

**Reply by WSzP** [reply]
thank you so much for your kind words. I agree that a browser extension would be great, and your dall-e one is awesome. Just not sure how to handle for a relatively complex case, such as promptomania, it with the fast-moving nature of diffusion models. But I would also love to collaborate. Hopefully we can make something great.

**Reply by altryne** [reply]
Would be so awesome, I have found a UI designer that's really eager to continue working on the extension, inject even more styles and modifiers. Is there a github repo of your own where we can chat about potential collaboration?

**Reply by ABBEN4** [reply]
If you guys want to wrap that up in an android / IOS app, we could figure something out. I can also assist on prompt engineering to make presets, been playing with that for a while now !

**Comment by ABBEN4** [comment]
Hello ! Abderrahim Benmoussa here, daytime researcher in cancer, rest of the time AI-art / digital painting / AI enthusiast working on UX for some projects, also building-up a free prompt builder. Will be glad to help the best I can !

**Comment by JoshuaKimsey** [comment]
Greetings everyone!

I'm Joshua Kimsey, I've been a Linux user in some form for over a decade now, and been using it exclusively for going on 3 1/2 years. I'm also CompTIA Linux+ certified. Always have found AI/ML to be fascinating, and with the growth of this field into the open-source domain, my interest has only grown.

I currently am maintaining a [Guide](https://rentry.org/linux-sd) and Bash [Script](https://github.com/JoshuaKimsey/Linux-StableDiffusion-Script/blob/main/linux-sd.sh) that automates the process of installing, updating, and running hlky's fork of Stable Diffusion for Linux users.

I absolutely love what has developed just in the past week, and I want to see where everything goes in the wild new world. :)

**Reply by hlky** [reply]
Thanks, optimized img2img is working now btw

**Reply by hlky** [reply]
https://github.com/hlky/stable-diffusion/wiki/Linux-Installation

**Reply by hlky** [reply]
There's also a Dockerfile now, which is compatiable with both Windows and Linux, not sure if there needs to be a guide for it as I think docker is kind of advanced usage for most people, and if someone knows how to use docker they should be able to just use it

Whereas Linux does have users of different skill levels these days, so a guide is useful for some

**Reply by JoshuaKimsey** [reply]
Awesome! I'll update my guide shortly. Doing a bit more clean up on the script, will update the guide once that is pushed. 

Also, thanks for the wiki page for the guide! :)

**Reply by JoshuaKimsey** [reply]
And yeah, if someone knows how to use Docker, it should pretty easy for them to know what to do. I can put a link to that in the guide though, so others can know.

**Reply by Kyxsune** [reply]
I literally just found this thread, after stumbling upon the tweet and documentation. Awesome stuff :D

**Reply by hlky** [reply]
> tweet 

There's a tweet mentioning this repo?

**Reply by Kyxsune** [reply]
> > tweet
> 
> There's a tweet mentioning this repo?

Sorry for the delay. Here's the tweet: https://twitter.com/altryne/status/1563452704063901696?cxt=HHwWgMDSje38v7IrAAAA

Here's the thread that led me here: https://twitter.com/altryne/status/1563452692399214594 .

Hitting a no Module named 'frontend' on the Linux install script, will debug a little to make sure it isn't just me being a noob XD

**Reply by hlky** [reply]
>named 'frontend' on the Linux install 

Pull the repo again, there was an update a short while ago that refactored the entire UI

**Reply by hlky** [reply]
@altryne is a contributor on the dev repo btw :)

**Reply by altryne** [reply]
And I almost finished the colab!

**Reply by JoshuaKimsey** [reply]
@Kyxsune Yeah, you'll wanna choose the option to update the repo when the script first runs cause of the big update to the UI that happened. If you keep having issues, open a new discussion and I can help you out more so with it. :)

**Reply by Kyxsune** [reply]
Thanks for the heads up, turns out I was running the shell with elevated perms so the conda installation had a few conflicts 😅. Environment still being built, but I'll follow up if anything weird happens :D

**Comment by altryne** [comment]
Hey folks, I'm @altryne , one of the maintainers (@hlky can I be added here as well as the dev repo?) 

I worked on the webui refresh, and just now ported our work into a colab! 

https://github.com/altryne/sd-webui-colab

Please open issues, submit comments, and help test out, the speed with which @hlky is moving is incredible!

**Comment by Kyxsune** [comment]
Hey All,

I'm @Kyxsune. I'm a bioinformatician working in drug design and a generalist by "evolution" (Front end, backend, and middleware). 
I only have a few years of experience in this area, but I want to assist in any way I can :D.

**Comment by ledadu** [comment]
Hi,
the --share was disabled ! in this commit by @hlky : 
https://github.com/hlky/stable-diffusion/commit/40184408c35551cf2bad16679819baee8a9f6968

and the --share {vars} of @altryne colab not work anymore..

**Reply by altryne** [reply]
@hlky ??

Colab doesn't work without this and honestly my whole set up for which I'm workin on this doesn't work.

**Reply by hlky** [reply]
You didn't tell me that's why you added it, I already reverted it

**Reply by altryne** [reply]
haha I was just adding a pin to the colab 🤣 
https://github.com/altryne/sd-webui-colab/commit/17cfcf4effd5a9697fc75bb929aa7aaaa90f84b9

**Reply by altryne** [reply]
@hlky let's find a way to chat between us (signal?)

**Reply by hlky** [reply]
I don't have signal, I can get it, I followed you on twitter so can use that for now if you want.
